ProcessOptimizer/model_systems/gold_map.py

```diff
@@ -0,0 +1,27 @@
+from typing import List
+import numpy as np
+
+from . import ModelSystem
+
+
+def score(coordinates: List[int]):
+    """Modified Branin-Hoo function. It has three approximate local minima.
+
+    It returns a value between 0 and -3.1, interpreted as the negative value of
+    amount of gold found at the given coordinates, in mg. The negative value is
+    to make it play nicely together with ProcessOptimizer, which is a minimizer.
+
+    More details: <http://www.sfu.ca/~ssurjano/branin.html>
+    """
+    x = 10 - coordinates[0]
+    y = coordinates[1]
+    gold_found = ((y - 1 / 8 * x**2 + 1.6 * x - 6) ** 2 + 10 * np.cos(x) - 299) / 100
+    return gold_found
+
+
+gold_map = ModelSystem(
+    score,
+    space=[(0.0, 15.0), (0.0, 15.0)],
+    noise_model="zero",
```

Comment:
```suggestion
    noise_model=None,
```
We don't have to use `None`, I would just like us to consistently use any of `None, "none" or "zero"` in across all the model system versions where we don't add noise. I have suggested `None` across the board.

Reply:
Yeah, I was getting cute with the different ways to do that. I'll add tests for each of them instead.